ovos_utils/dialog.py

```diff
@@ -0,0 +1,193 @@
+import os
+import random
+import re
+from os.path import join
+from pathlib import Path
+
+from ovos_utils.bracket_expansion import expand_options
+from ovos_utils.configuration import read_mycroft_config
+from ovos_utils.file_utils import resolve_resource_file
+from ovos_utils.lang import translate_word
+from ovos_utils.log import LOG
+
+
+class MustacheDialogRenderer:
```

Comment:
Am I correct in assuming that `expand_options()` is the only functional change from the Mustache in Mycroft-core?

If so, seeing this pass against HolmesV would be sufficient.